include/controller/cpp/webots/Node.hpp

```diff
@@ -126,8 +126,12 @@ namespace webots {
     Node *getParentNode() const;
     bool isProto() const;
     Node *getFromProtoDef(const std::string &name) const;
+    int getFieldCount() const;
```

Comment:
I would rename this function
```suggestion
int Node::getNumberOfFields() const {
```

same for `getProtoFieldCount`